post-processor/shell-local/communicator_test.go

```diff
@@ -33,9 +57,9 @@ func TestCommunicator(t *testing.T) {
 
 	cmd.Wait()
 
-	if cmd.ExitStatus != 0 {
-		t.Fatalf("err bad exit status: %d", cmd.ExitStatus)
-	}
+	// if cmd.ExitStatus != 0 {
+	// 	t.Fatalf("err bad exit status: %d", cmd.ExitStatus)
```

Comment:
let's make sure we test this

Reply:
Where do you want to remove the {{.Vars}} template variable?  p.Config.Vars is still being used in the communicator directly by exec.Command.

Reply:
not sure I follow

Reply:
oh I see, this was a response to a comment above. I think I was referring to the docs

```
 -   `execute_command` (string) - The command to use to execute the script. By
    default this is `chmod +x "{{.Script}}"; {{.Vars}} "{{.Script}}"`.
    The value of this is treated as [template engine](/docs/templates/engine.html).
    There are two available variables: `Script`, which is the path to the script
    to run, `Vars`, which is the list of `environment_vars`, if configured.
```

Reply:
Sorry, was replying to an outdated diff.

Reply:
yeah you got it.